benchmark/runner.ts

```diff
@@ -7,7 +7,7 @@ export async function runBenchmark<T, R>({
   id,
 }: {
   prepareTest: (i: number) => PromiseOptional<{input: T; resultCheck?: (result: R) => boolean}>;
-  testRunner: (input: T) => PromiseOptional<R>;
+  testRunner: (input: any) => PromiseOptional<R>;
```

Comment:
Please don't remove typings, that affects negatively the type-safety of all code not just the noble benchmarks

Reply:
Okay, it's hard to me to understand what's going on here, typings are not too helpful. I usually don't add them to tests / benchmarks. Maybe you could fix this

Reply:
You have to update the types here https://github.com/ChainSafe/bls/blob/86c956e12a3026a1226f1fbcedd234ea8be60a0f/benchmark/noble.ts#L10